src/colmap/controllers/incremental_pipeline_test.cc

```diff
@@ -73,6 +73,56 @@ void ExpectEqualReconstructions(const Reconstruction& gt,
   }
 }
 
+void ExpectDifferentReconstructions(const Reconstruction& gt,
```

Comment:
Can this be simplified by using the inverse logic of the ExpectIdenticalReconstructions ?